### PR TITLE
Implement wasm-debug extension

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Add 'wabt' tool to analyze wasm file
- Add 'wasm -> wast' menu to translate file from wasm to wast
- Move 'convertRelativePath' function to the brackets util

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: libs/brackets-server/embedded-ext/wasm-debug/nls/root/strings.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// English - root strings
+define({
+    "ERROR_TRANSLATE_WASM"    : "Failed to translate wasm!",
+    "ERROR_NOT_FOUND_FILE"    : "Not found selected file",
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/wasm-debug/nls/root/strings.js]: check english for "Not found selected file" ?

File: libs/brackets-server/embedded-ext/wasm-debug/nls/ko/strings.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// Korean - ko strings
+define({
+    "ERROR_TRANSLATE_WASM"    : "WASM을 변환하는데 실패했습니다!",
+    "ERROR_NOT_FOUND_FILE"    : "선택된 파일이 없습니다.",
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/wasm-debug/nls/ko/strings.js]: may be, ERROR_NOT_FOUND_FILE -> ERROR_FILE_NOT_FOUND looks better ?

File: libs/brackets-server/embedded-ext/wasm-debug/nls/root/strings.js (hunk 1 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/wasm-debug/nls/root/strings.js]: Fixed.

File: libs/brackets-server/embedded-ext/wasm-debug/nls/ko/strings.js (hunk 2 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/wasm-debug/nls/ko/strings.js]: Fixed.

File: libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                callback(err);
+            });
+        }
+
+        _domainManager.registerCommand(
+            "debugDomain",
+            "wasm2wast",
+            handleWasm2Wast,
+            true,
+            "Translate wasm to wat",
```
Comment (hyundukkim) [inline libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js]: Need to fix from 'wat' -> 'wast'

Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js]: Fixed. Thanks.

File: launch [hunk 4]
```diff
@@ -546,7 +586,11 @@ def main():
     if emscripten_path == '': return 1
 
   Log.info('Use emscripten at %s' % emscripten_path)
-  pathlist = [grunt_path, emscripten_path, os.path.join(tool_path, 'WebIDLBinder'), '']
+
+  if emscripten_path != '':
```
Comment (Kwanghyuk-Kim) [inline launch]: IMO, this line is unnecessary as emscripten_path already checked on above.

File: libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                callback(null);
+            });
+
+            child.on("error", (err) => {
+                callback(err);
+            });
+        }
+
+        _domainManager.registerCommand(
+            "debugDomain",
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js]: debugDomain string seems to be used repeatly.
What about using string definision or variable ?

File: launch (hunk 4 above)
Comment (HunseopJeong) [reply launch]: I agreed. Fixed.

File: libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js (hunk 5 above)
Comment (HunseopJeong) [reply libs/brackets-server/embedded-ext/wasm-debug/node/DebugDomain.js]: Done.

**Comment by HunseopJeong** [comment]
Free to merge :)

**Pull request merged**